tests/python/examples/algorithm/test_laplace.py

```diff
@@ -0,0 +1,11 @@
+from taichi.examples.algorithm.laplace import *
```

Comment:
shall we just import laplace here?

Reply:
We can `from taichi.examples.algorithm import laplace`, but then we have to add `laplace.` prefix to all variables and functions. Since the test is exclusively designed for the corresponding example, is it a bit tedious to add that?

Reply:
@strongoier  oh I meant we can do `from taichi.examples.algorithm.laplace import laplace` ;)

Reply:
Ah yes. Updated.